database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/LoginView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Copyright (c) 2015 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+struct LoginView: View {
+
+  @ObservedObject var user: UserManager
+  var screenWidth = UIScreen.main.bounds.width
+  var screenHeight = UIScreen.main.bounds.height
+
+  var body: some View {
+    NavigationView {
+      VStack {
+
+        // Login title
+        Text("LOGIN")
+          .font(.title)
+
+        Spacer()
+          .frame(idealHeight: 0.1 * screenHeight)
+          .fixedSize()
+
+        // Email textfield
+        HStack {
+          Image("user-icon")
+            .resizable()
+            .scaledToFit()
+            .frame(width: 30.0, height: 30.0)
+            .opacity(0.5)
+          TextField("Email", text: $user.email)
```

Comment:
re: the bug where the textfields were not being emptied upon logging in:  
Since `UserManager` is a reference type and we maintain a strong reference to it in `ContentView`, I'm wondering if that's what's keeping the textfields from emptying because the `UserManager` object is still alive and has a non-nil `user.email` and `user.password` from the sign in process. 
A debugging idea:  try conditionally populating the textfields with `$user.email` if no user is signed in and `""` if a user is signed in.